### PR TITLE
Edit Item Form Modal

### LINKED ISSUE #33: Create : Profile Page - Profile Item Card Edit Form
## Summary

When the user clicks on edit on an item on his profile, he must be able to edit it with a form. Create a controlled form with validation. 

User must not be able to change the price, its unethical. 

Design and responsiveness must be readied with Tailwind, snapshot/unit tests should also be written.

### Screenshots

N/A

### Basic example

N/A

### Motivation

To let user be able to edit the item that they put to sale/donation.

### PR DESCRIPTION
**Pull request by sarahaboudalal**

### CODE REVIEW

**Review by OussamaMasri — APPROVED** [review]
Great work Sarah!

**Review by ayubabdullah — APPROVED** [review]
Good work sara 😎👌

File: src/components/ItemEditForm/EditItemModal.jsx [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+  'Kids',
+  'Toys',
+  'Books',
+  'Sports',
+  'Clothes',
+  'Furniture',
+  'Electronics',
+  'Homemade',
+  'Other',
+];
```
Comment (Iseluin) [inline src/components/ItemEditForm/EditItemModal.jsx]: This is iterating in both the new modal and our currently merged form, can you move these inside a "constants" file in src/utils in both components, so DRY code.

**Review by Iseluin — CHANGES_REQUESTED** [review]
Nice job! Can you check the comment and adjust your code ? Also, it would be nice if we would let users to close the modal with an "X" too and by clicking outside the modal.

**Comment by haboosh92** [comment]
Great work , very nice design and clean code

**Review by haboosh92 — APPROVED** [review]
Great jog

**Review by Iseluin — APPROVED** [review]
Noice 👍🏻

**Pull request merged**